.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -43,25 +43,8 @@ body:
       description: Please run the following script and paste the output
       value: |
         ```python
-        import astropy
-        try:
-            astropy.system_info()
-        except AttributeError:
-            import platform; print(platform.platform())
-            import sys; print("Python", sys.version)
-            import astropy; print("astropy", astropy.__version__)
-            import numpy; print("Numpy", numpy.__version__)
-            import erfa; print("pyerfa", erfa.__version__)
-            try:
-                import scipy
-                print("Scipy", scipy.__version__)
-            except ImportError:
-                print("Scipy not installed")
-            try:
-                import matplotlib
-                print("Matplotlib", matplotlib.__version__)
-            except ImportError:
-                print("Matplotlib not installed")
+        from astropy.utils import system_info
+        system_info()
```

The suggested code does not work. You can just use the original code. Thanks.
```suggestion
        import astropy
        astropy.system_info()
```